CONTRIBUTING-TO-ESAPI.txt

```diff
@@ -47,8 +48,7 @@ Overview:
 
 Required Software:
     We use Maven for building. Maven 3.3.9 or later is required. You also need
-    JDK 8 or later.
-    [Note: If you use JDK 9 or later, there will be multiple
+    JDK 8 or later. [Note: If you use JDK 9 or later, there will be multiple
```

Comment:
@noloader - I think you got this to work with Java 11, right? And submitted a PR for that? If so, we should adjust this warning for JDK 9 or later.